### PR TITLE
OCPQE-22055 bump ec to rc for 4.16

### PR DESCRIPTION
**Pull request by dis016**
Re opening https://github.com/openshift/release/pull/52084 to resolve the merge conflicts.

### CODE REVIEW

**Comment by dis016** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.16-azure-ipi-proxy-tp-f28

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-stable.yaml [hunk 1]
```diff
@@ -99,4 +99,4 @@ zz_generated_metadata:
   branch: release-4.16
   org: openshift
   repo: openshift-tests-private
-  variant: amd64-rollback-ec
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-stable.yaml:102]: Also need update the line like https://github.com/openshift/release/pull/52084/commits/421132164f2872ebfa66b7175cb53d1a94503d24#diff-8de710535f0a5755e0356538df0e71e1d94eb52274b6896749e96741bb5aff1cL71, right?

Comment (dis016) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-stable.yaml:102]: @jianlinliu yes, good catch. Thanks

Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-stable.yaml:102]: The last comment is not resolved yet, right?

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-stable.yaml:102]: @dis016 Do we need to remove `stream: 4-dev-preview` in this file, too ?

Comment (dis016) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-stable.yaml:102]: @liangxia updated.

**Comment by jianlinliu** [comment]
@dis016 ci/prow/core-valid  failed because conflicts, need a rebase to resolve it.

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml [hunk 2]
```diff
@@ -33073,8 +33075,100 @@ periodics:
       secret:
         secretName: result-aggregator
 - agent: kubernetes
+<<<<<<< HEAD
   cluster: build02
   cron: 13 18 10 * *
+=======
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml:33081]: Looks like something wrong when rebase

Comment (dis016) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml:33081]: resolved in latest commit

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml [hunk 3]
```diff
@@ -32363,6 +32363,8 @@ periodics:
 - agent: kubernetes
   cluster: vsphere02
   cron: 14 12 29 * *
+  cluster: build05
```
Comment (jianlinliu) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml:32366]: Sounds introduced incorrectly when resolving rebase conflicts.

Comment (jianlinliu) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml:32366]: I suggest to re-run `make jobs`.

**Pull request merged**